src/test/java/cz/upce/fei/inptp/zz/entity/SchoolTest.java

```diff
@@ -206,4 +206,38 @@ public void testRemovedStudentWasRemovedFromHisCourseAction(){
        
         assertEquals(0, courseAction.getStudents().size());
     }
+    
+    @Test
+    public void testHasCapacityOnEmptyCourseAction() {
```

Comment:
All these tests are only CourseAction unit tests and shouldn't been placed in SchoolTest class

Reply:
Solved.